docs/wifi-setup.md

```diff
@@ -18,8 +18,112 @@ There are several ways to connect your Anthias instance to Wi-Fi. You can either
 configure the Wi-Fi settings via the captive portal or you can run the `raspi-config`
```

Comment:
Let's discourage the use of `raspi-config` WiFi config, since it will interfere with how the captive portal works.

Reply:
On it, thanks.

Reply:
All done. (I removed the `raspi-config` section.) Requesting another review, @vpetersson... Thanks!